### ISSUE TITLE
Submission Errors Give fields not GUI Names

### ISSUE THREAD

**Issue by emmahodcroft** [loculus-project/loculus#2558]
For the Current Ebola Zaire sample data, I get a double error: `geoLocCountry` and `sampleCollectionDate` are required.
![image](https://github.com/user-attachments/assets/74c7d4b3-220a-4bfc-847d-1b569490fcf7)

However, when I go to edit the record, none of the fields are called like this - they are all renamed for the GUI:
![image](https://github.com/user-attachments/assets/a2ab12d3-6711-4d40-87a4-3a6dcdc4ca6e)
![image](https://github.com/user-attachments/assets/7db4dccc-f69a-4d77-889e-0ce376fc3357)

In this case it's not _too_ hard to figure out the fields it means, but this is not great practice, and in some cases it could be much harder (esp if number of required fields is higher and involves multiple dates or locations). 

I'd suggest the original error use whatever the GUI will show later. 

Another improvement would be to highlight in red or similar any/all fields that are required and currently empty.

**Comment by emmahodcroft** [comment]
Is this fixed in the recent edits to the review page, @fhennig ? (I think so but checking)

**Comment by corneliusroemer** [comment]
I don't think this is fixed, all our errors are still calculated on the basis of output fields which is confusing for users. We should switch to an input-field based model which is much more intuitive for users.

@anna-parker @theosanderson and I should look into https://github.com/loculus-project/loculus/pull/932 - this predates Anna and I think she could have valuable thoughts as well.
